### PR TITLE
Streamline repos contents, follow conventions, AI prompts

### PR DESCRIPTION
**Pull request by vmelamed**
# Summary

## Commits

- chore: streamline repos contents, follow conventions, AI prompts

## Checklist

- [x] Commit messages follow [Conventional Commits](https://www.conventionalcommits.org/) format
- [x] Tests pass locally, and cover new code and edge cases. Test coverage is above 80%.
- [ ] It is not expected that the performance will degrade by more than 20%
- [x] Documentation is updated if necessary

### CODE REVIEW

File: .github/CONVENTIONS.md [hunk 1]
```diff
@@ -67,302 +68,215 @@ The project owner is a non-native English speaker.
...
+  - `src/` — source code for deliverables
+  - `tests/` — test projects (xUnit v3, FluentAssertions, NSubstitute, MTP v2, Coverlet)
+  - `benchmarks/` — BenchmarkDotNet projects (desirable)
+  - `examples/` — usage examples (desirable; prefer single-file programs)
+  - `docs/` — documentation beyond README.md (optional)
+  - `.github/`
+    - AI-specific guidance and conventions (`CONVENTIONS.md` and/or `copilot-instructions.md`)
+    - dependabot configuration (`dependabot.yml`)
+    - issue and pull request templates (`ISSUE_TEMPLATE/` and `PULL_REQUEST_TEMPLATE.md`)
+  - `.github/workflows/` — GitHub Actions CI/CD (`CI.yaml`, `Prerelease.yaml`, `Release.yaml`, `RefreshLockFiles.yaml`, `AutoMerge.yaml`, `ClearCache.yaml`, `RebuildBenchHistory.yaml`)
```
[the remark this answers is not in the document]
Comment (vmelamed) [reply .github/CONVENTIONS.md:94]: directory

File: .github/copilot-instructions.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
+
+@~/.claude/CLAUDE.md
+@~/repos/vm2/CLAUDE.md
+@.github/CONVENTIONS.md
+@docs/ARCHITECTURE.md
+@docs/WORKFLOWS_REFERENCE.md
+@docs/GIT_PLAYBOOK.md
+@docs/RELEASE_PROCESS.md
+
```
[the remark this answers is not in the document]
Comment (vmelamed) [reply .github/copilot-instructions.md:23]: removed

File: .github/copilot-instructions.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
+# Copilot Instructions for vm2.MyPackage
```
[the remark this answers is not in the document]
Comment (vmelamed) [reply .github/copilot-instructions.md:1]: changed

File: .github/copilot-instructions.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+               └───────► _ci.yaml ─┬───────────────────────► validate-commits.sh ───────────►
+                                   ├───────────────────────► validate-inputs.sh ────────────►
+                                   ├──► _build.yaml ───────► build.sh ──────────────────────►
+                                   ├──► _test.yaml ────────► run-tests.sh ──────────────────►
+                                   ├──► _benchmarks.yaml ──► run-benchmarks.sh ─────────────►
+                                   └──► _pack.yaml ────────► pack.s ────────────────────────►
+
+Prerelease.yaml ───────► _prerelease.yaml ───────────────┬─► compute-prerelease-version.sh ─►
+                                                         ├─► changelog-and-tag.sh ──────────►
+                                                         └─► publish-package.s ─────────────►
```
[the remark this answers is not in the document]
Comment (vmelamed) [reply .github/copilot-instructions.md:57]: edited

File: .github/copilot-instructions.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
+Release.yaml ──────────► _release.yaml ──────────────────┬─► compute-release-version.sh ────►
+                                                         ├─► changelog-and-tag.sh ──────────►
+                                                         └─► publish-package.sh ────────────►
+
+```
+
+See `$VM2_REPOS/vm2.DevOps/docs/ARCHITECTURE.md` for the full design and `$VM2_REPOS/vm2.DevOps/docs/WORKFLOWS_REFERENCE.md` for workflow inputs/outputs.
```
[the remark this answers is not in the document]
Comment (vmelamed) [reply .github/copilot-instructions.md:65]: edited

File: .github/copilot-instructions.md [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+| `_dotnet.sh`      | Manages the output of `dotnet build` command                     |
+| `_dump_vars.sh`   | Dumps the values of bash variables in a tabular format           |
+| `_user.sh`        | User interface primitives                                        |
+
+See `scripts/bash/lib/FUNCTIONS_REFERENCE.md` for the full function inventory.
+
+## Shared File Sync
+
+Files that are canonical in `vm2.Templates` (`.editorconfig`, `.github/CONVENTIONS.md`, `Directory.Build.props`, etc.) are
+synced here via `diff-shared.sh`. The mapping is in `scripts/bash/diff-shared.config.json`.
```
[the remark this answers is not in the document]
Comment (vmelamed) [reply .github/copilot-instructions.md:138]: fixed
